cms/settings.py

```diff
@@ -79,8 +80,10 @@
         "APP_DIRS": True,
         "OPTIONS": {
             "context_processors": {
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
                 "django.contrib.auth.context_processors.auth",
-                "django.template.context_processors.static",
+                "django.contrib.messages.context_processors.messages",
```

Comment:
What's this do?

Reply:
It's required by the message middleware, which I'm not certain is being used anywhere besides the admin panel